docker/llm/finetune/lora/cpu/docker/bigdl-lora-finetuing-entrypoint.sh

```diff
@@ -1,46 +1,66 @@
 #!/bin/bash
 set -x
-source /opt/intel/oneapi/setvars.sh
-export CCL_WORKER_COUNT=$WORLD_SIZE
-export CCL_WORKER_AFFINITY=auto
-
-if [ "$WORKER_ROLE" = "launcher" ]
+if [ "$STANDALONE_DOCKER" = "TRUE" ]
 then
-  sed "s/:1/ /g" /etc/mpi/hostfile > /home/mpiuser/hostfile
-  export DATA_PATH="/ppml/data/$DATA_SUB_PATH"
-  sleep 10
+  source /opt/intel/oneapi/setvars.sh
+  export CCL_WORKER_COUNT=2
+  export CCL_WORKER_AFFINITY=auto
+  export MASTER_ADDR=172.18.0.3
```

Comment:
set `MASTER_ADDR` with the ip address of the container, which can be got by hostname command